pants.ci.aarch64.toml

```diff
@@ -0,0 +1,8 @@
+[GLOBAL]
+# Our physical aarch64 CI instance is a very burly machine with 80 cores.
+# A single GHA self-hosted runner can only run one job at a time, so to use
+# more of the machine's resources, we run multiple self-hosted runners on it
+# and constrain each one here so they don't all try and use all the cores.
+# TODO: Control this externally, on the machine itself?
+rule_threads_core = 8
```

Comment:
It shouldn't actually be necessary to set these (we don't in CI by default), because their values are auto-detected.

Are the values poorly auto-detected in the relevant environment? If so, should probably mention that here instead, and possibly open a ticket about it.

Reply:
They would be autodetected relative to "all the cores on the machine", whereas we want to restrict to fewer cores than that because we have multiple runners on the machine. That is what the comment was trying to say...

Reply:
Ah. And they're not containerized?